tools/bisector/bisector/bisector.py

```diff
@@ -5138,10 +5249,13 @@ def _main(argv):
 
     SHOW_TRACEBACK = args.debug
     service_hub = ServiceHub()
+
     try:
-        service_hub.register_service('upload', ArtifactorialService())
+        artifactorial = ArtifactorialService()
+        service_hub.register_service('upload', artifactorial)
+        service_hub.register_service('download', artifactorial)
```

Comment:
I guess we want to add ArtifactoryService here too. We could try ArtifactoryService first and if it raises an exception then we register the other one. This just means we have to check for the expected env var in `ArtifactoryService.__init__` and raise e.g. `ValueError` if they are not present.